### PR TITLE
chore: remove duplicate and un-needed files from dist zips

### PR DESCRIPTION
**Pull request by MarshallOfSound**
Notes: no-notes

### CODE REVIEW

**Comment by MarshallOfSound** [comment]
@jkleinsc Although this successfully removed duplicate files from the zip file, it looks like the ignore path didn't work

image

Will fix that tomorrow before merging this 👍

**Comment by codebytere** [comment]
(approved pending above fix)

File: build/zip.py [hunk 1]
```diff
@@ -57,7 +61,8 @@ def main(argv):
   with open(runtime_deps) as f:
     for dep in f.readlines():
       dep = dep.strip()
-      dist_files += [dep]
+      if dep not in dist_files:
+        dist_files += [dep]
```
Comment (nornagon) [inline build/zip.py]: better: just make `dist_files` a set.

File: build/zip.py [hunk 2]
```diff
@@ -5,9 +5,11 @@
 import zipfile
 
 LINUX_BINARIES_TO_STRIP = [
+  'chromedriver',
   'electron',
   'libffmpeg.so',
-  'libnode.so'
+  'libnode.so',
+  'mksnapshot'
```
Comment (nornagon) [inline build/zip.py]: these binaries shouldn't be stripped as part of the zip process, i don't think. i missed the original review that added this though :(

Comment (nornagon) [reply build/zip.py]: yes, i agree that these binaries should be zipped, i just don't think that `zip.py` is the right place to do it. I'll reserve further comment until i have a concrete suggestion though :)

Comment (nornagon) [reply build/zip.py]: oops, s/zipped/stripped in last comment

File: build/zip.py (hunk 1 above)
Comment (MarshallOfSound) [reply build/zip.py]: I wasn't sure if `dist_files` needed to maintain a certain order that set's don't guarantee

cc @jkleinsc ❓

**Comment by MarshallOfSound** [comment]
@jkleinsc Looks like we're stripping non-arm binaries on arm platforms (`mksnapshot`).  Might have to add a special case for that

File: build/zip.py (hunk 1 above)
Comment (jkleinsc) [reply build/zip.py]: @MarshallOfSound I don't think order matters

**Comment by jkleinsc** [comment]
@MarshallOfSound yeah... I saw that.  I think I'm going to pull the stripping changes out of this PR and address in another PR.

**Comment by jkleinsc** [comment]
Merging as linux-x64-testing-tests was resolved in #14875.

File: build/zip.py (hunk 1 above)
Comment (nornagon) [reply build/zip.py]: yes, order doesn't matter.

**Pull request merged**